### PR TITLE
Website Command

### PR DESCRIPTION
**Pull request by BlackBearFTW**
Added a website command

?website [optional argument]

### CODE REVIEW

File: src/commands/WebsiteCommand.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+			"Displays the website with given parameter",
+			{
+				aliases: ["web"]
+			}
+		);
+	}
+
+	async run(message: Message, args: string[]) {
+		if (args[0] === undefined || args[0] === null) args[0] = "";
+		await message.channel.send(`https://codesupport.dev/${args[0]}`);
```
Comment (LamboCreeper) [inline src/commands/WebsiteCommand.ts]: You could shorten this entire check to just `${args[0] || ""}`

File: src/commands/WebsiteCommand.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import {Message} from "discord.js";
+import Command from "../abstracts/Command";
+
+class WebsiteCommand extends Command {
+	constructor() {
+		super(
+			"website",
+			"Displays the website with given parameter",
```
Comment (LamboCreeper) [inline src/commands/WebsiteCommand.ts]: Can this please be `Displays a link to the CodeSupport's website (or a specific page if specified).`

**Review by LamboCreeper — CHANGES_REQUESTED** [review]

File: src/commands/WebsiteCommand.ts (hunk 1 above)
Comment (BlackBearFTW) [reply src/commands/WebsiteCommand.ts]: 👍

File: src/commands/WebsiteCommand.ts (hunk 2 above)
Comment (BlackBearFTW) [reply src/commands/WebsiteCommand.ts]: 👍

**Comment by BlackBearFTW** [comment]
Done 😄

**Review by LamboCreeper — APPROVED** [review]

**Pull request merged**